macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -44,7 +44,9 @@
             edr_bucket_end as bucket_end,
             {{ elementary.timediff("hour", "edr_bucket_start", "edr_bucket_end") }} as bucket_duration_hours,
             {{ elementary.null_string() }} as dimension,
-            {{ elementary.null_string() }} as dimension_value
+            {{ elementary.null_string() }} as dimension_value,
+            {{ elementary.quote(timestamp_column) }} as config__timestamp_column,
```

Comment:
I think the `config__` isn't really necessary and that we can just use `timestamp_column` and `where_expression` as the column names. What do you think?

Reply:
I don't feel strongly either way - so changing to remove the `config__`

Reply:
done